Documentation/reference/f5-ltm-lb.md

```diff
@@ -0,0 +1,92 @@
+# Using F5 BIG-IP LTM with Tectonic
+
+Installing Tectonic on bare metal or VMWare requires a master DNS (which resolves to any master node in the cluster), and a worker DNS (which resolves to any worker nodes for Ingress). Evaluation clusters may be set up using a DNS entry that points to each node, but production clusters require the use of a load balancer.
+
+When installing on cloud providers such as AWS, Tectonic automatically creates a load balancer (such as AWS ELB). When installing Tectonic on bare metal or VMWare, you must provide your own load balancer.
+
+This document describes setting up F5 BIG-IP LTM for use as a load balancer with Tectonic.
+
+## Overview
+
+### Prerequisites
+
+* F5 BIG-IP LTM 10.x or higher
+    * In the same datacenter as the Tectonic cluster
+* Tectonic installer
+    * VMware or bare metal environment
+* Pre-allocated IP addresses for cluster and pre-created DNS records
+
+## DNS and IP address allocation
+
+Tectonic installation requires static allocation of IP Addresses for bare metal or VMWare. Create these required DNS records before launching Tectonic Installer.
+
+The following table lists an example Tectonic node, Kubernetes API node, and two master and two worker nodes.
+
+| Record | Type | Value |
+|------|-------------|:-----:|
+|tectonic.example.net | A | 192.168.1.91 |
+|api.example.net | A | 192.168.1.92 |
+|master-01.example.net | A | 192.168.1.110 |
+|master-02.example.net | A | 192.168.1.111 |
+|worker-01.example.net | A | 192.168.1.112 |
+|worker-02.example.net | A | 192.168.1.113 |
+
+In this example the virtual IP for console traffic will be 192.168.1.91, DNS resolvable at tectonic.example.net and the virtual IP for API traffic will be 192.168.92, DNS resolvable at api.example.net
+
+## Create F5 LTM Pools
+
+First, use the F5 web console to create two LTM pools for directing traffic to Tectonic console and API server.
+
+Create an F5 LTM Pool for the API Server:
+1. Log in to F5 BIG-IP LTM web console as an admin user.
```

Comment:
Again, I've learned to dislike this markdown numbering style. Altho it works in output, it doesn't for people reading the source directly, etc, and can be confusing for future editors.

Reply:
ah. ok.